cogs/secretSanta.py

```diff
@@ -0,0 +1,145 @@
+import nextcord
+from nextcord.ext import tasks, commands
+import time
+from datetime import *
+import random
+
+msg = ''
+msg_id = ''
+participatingList = []
+b = ''
+santas =[] 
+guild_id = 0
+messagedFlag = False
+
+class santa(commands.Cog):
+	def __init__(self, client):
+		self.client = client
+

```

Comment:
See previous comment on definining variables outside of the class. This method is the same without declaring variables in global. However, you will have to refer to them using self.
```suggestion
self.msg = ''
self.msg_id = ''
self.participatingList = []
self.b = ''
self.santas =[] 
self.guild_id = 0
self.messagedFlag = False
```